v2/futures/order_place_service_ws.go

```diff
@@ -0,0 +1,267 @@
+package futures
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/adshao/go-binance/v2/common"
+)
+
+// OrderPlaceWsService creates order
+type OrderPlaceWsService struct {
+	c      wsClient
+	signFn func(string, string) (*string, error)
+}
+
+// NewOrderPlaceWsService init OrderPlaceWsService
+func NewOrderPlaceWsService(apiKey, secretKey string) (*OrderPlaceWsService, error) {
+	conn, err := newConnection()
```

Comment:
Whether synchronous or asynchronous, the same connection is required, which is a long connection in WebSocket.  Is this the current situation where all these similar interfaces share a common connection. is that right?

Reply:
No it's not. Due to limited time I created WriteSync method that using the same connection but leaved comment for this. If you propose to have separate client with it's own connection, I agree. But It will takes sometime to implement

Reply:
Current scheme is 
service->client(sync && async)->single connection

OrderPlaceService and OrderCancel will have different connections 
I think it's ok to not have only one connection for all services, 
It's easier and more fault tolerant.

Reply:
I know, but the biggest advantage of WebSocket is sharing long connections. If you use a separate connection, I think it's better to use the REST interface. This is also why some people use WebSocket. Please take some time to implement this, it's almost done. @vigodsky